bonus_guides/D_mix_tasks.md

```diff
@@ -461,6 +461,12 @@ We can now do as the task suggests and inspect the contents of `priv/static` dir
 * a compressed file with gzip
 * a file containing the original file name and its digest
 * a compressed file containing the file name and its digest
+ 
+If you want to determine which files should be gzipped, you can use the application environment `:gzippable_exts` in config file:
```

Comment:
This is definitely noteworthy. Could we change so that it matches the style of the rest of the file?

Something like:

 > We can optionally determine which files should be gzipped, by using the `:gzippable_exts` option in the config file: